content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,76 @@
+---
+slug: 38
```

Comment:
38 is taken already: #512
The next free one is 41, afaik.

Reply:
Taken 41.